docs-ref-conceptual/azure-cli-extension-alias.md

```diff
@@ -95,7 +95,7 @@ az ls-groups
 
 ## Create an alias command with arguments
 
-You can also add positional arguments to an alias command by including them as `{{ arg_name }}` in the alias name. The whitespace inside the curly braces is required.
+You can also add positional arguments to an alias command by including them as `{{ arg_name }}` in the alias name. The whitespace inside the curly brackets is required.
```

Comment:
Remove change; `curly braces` is standard US-EN nomenclature for `{}`.

Reply:
Shouldn't that be just "braces" then? All braces are curly.

Reply:
Seems fine - go ahead and change to `braces`.

Reply:
Changed.